doc/zeek.rst

```diff
@@ -72,18 +72,19 @@ By default, *zkg* will install the most recent release version of the
 plugin. If you want to install the current development version, use
 ``zkg install --version main zeek/spicy-plugin`` instead.
 
+.. _zeek_spicyz:
```

Comment:
Let's just either add an anchor to the _Package Installation_ section and reference that, or add a subsection here.

Reply:
I was actually wanting to link specifically to the spicyz part, it's for the plugin's README.